fonts/squarize.py

```diff
@@ -210,144 +243,164 @@ def initialize_glyphs():
-        width_debilis=88
-        width_deminutus=88
-        #width of a punctum inclinatum (we consider that the punctum inclinatum auctus has the same width
-        width_inclinatum=164
-        #width of a stropha (idem for the stropha aucta)
-        width_stropha=164
-        #with of the second (highest) note of a pes
-        width_high_pes=154
-        # the width of the punctum inclinatum deminutus (no need to add the width of a line)
-        width_inclinatum_deminutus=82
-        # the width of the note which is just before a deminutus, in some fonts it is not the same width as a punctum
-        width_flexusdeminutus=186
-        # the widths of the torculus resupinus, there are five, one per note difference between the two first notes (for example the first is the width of the first two notes of baba
-        porrectusflexuswidths=(340,428,586,670,931)
-        porrectuswidths=(490,575,650,740,931)
-        # width that will be added to the standard width when we will build horizontal episemus. for example, if a punctum has the length 164, the episemus will have the width 244 and will be centered on the center of the punctum 
-        hepisemus_additional_width=5
-    elif (font_name=="parmesan"):
-        base_height=157.5
-        line_width=22
-        width_punctum=161
-        width_oriscus=192
-        width_oriscus_rev=192
-        width_quilisma=161
-        width_debilis=75
-        width_deminutus=75
-        width_inclinatum=178
-        width_stropha=169
-        width_high_pes=151
-        width_inclinatum_deminutus=112
-        width_flexusdeminutus=161
-        porrectusflexuswidths=(340,428,586,670,931)
-        porrectuswidths=(490,575,650,740,931)
-        hepisemus_additional_width=5
-    elif (font_name=="greciliae"):
-        base_height=157.5
-        line_width=18
-        width_punctum=166
-        width_oriscus=166
-        width_oriscus_rev=168
-        width_quilisma=166
-        width_debilis=65
-        width_deminutus=65
-        width_inclinatum=185
-        width_stropha=163
-        width_high_pes=155
-        width_inclinatum_deminutus=139
-        width_flexusdeminutus=168
-        porrectusflexuswidths=(503,629,628,628,931)
-        porrectuswidths=(503,629,628,628,931)
-        hepisemus_additional_width=5
-    if (font_name=="gregoria"):
-        base_height=157.5
-        line_width=22
-        width_punctum=164
-        width_oriscus=164
-        width_oriscus_rev=164
-        width_quilisma=164
-        width_debilis=88
-        width_deminutus=88
-        width_inclinatum=173
-        width_stropha=164
-        width_high_pes=154
-        width_inclinatum_deminutus=128
-        width_flexusdeminutus=186
-        porrectusflexuswidths=(340,428,586,670,931)
-        porrectuswidths=(490,575,650,740,931)
-        hepisemus_additional_width=5
+        LINE_WIDTH = 22
```

Comment:
I'm affraid this (global variables) is quite anti-pattern... What should be done here is a `width` hash passed to the other functions. Quite intrusive but will be much better